### DISCUSSION TITLE
Add a New Folder/Directory on the Repository Page

### DISCUSSION THREAD

**Discussion by ginaghouri** [community/community#103164] [Repositories]
### Select Topic Area

Product Feedback

### Body

Please, Add a New Folder/Directory Button on the Repository Page to improve interface.

**Comment by gil-son** [comment]
To add folder you must to clone/remote your project and create a folder inside of your project. After that organize the commit e push your project to Github

**Comment by NipulM** [comment]
Hello 👋🏼, 

You can create a new folder or directory using two different methods.

**GitHub Web Interface:**
1. Go to your Repository, Click on "Add File" and Choose "Create New File".
2. Specify the Folder Path:
- In the "Name your file..." field, type the folder name followed by a forward slash ("/") to create a folder. For example, if you want to create a folder named "new-folder," type "new-folder/".
3. To create a folder you need to add at least one file to it. Therefore add a dummy file by typing. e.g: dummy.txt
4. Scroll down to the bottom of the page. In the "Commit new file" section, you need to provide a commit message. Enter a meaningful commit message indicating that you are adding a new folder. For example, "Add new-folder."
5. Click the "Commit new file" button.

**Git Command Line:**
1. create a New Directory:
`mkdir new-folder`

2. Navigate into the New Directory:
`cd new-folder`

3. Add a file:
`touch dummy.txt`

4. Stage and Commit changes
`git add .`
`git commit -m "Add new-folder with dummy file`

5. Push Changes to the main branch
`git push origin main`


If this helps, please mark this as Answered! :)

**Reply by ginaghouri** [reply]
I was talking about a button function I'm suggesting the developer to create like it's easy to upload files to a repo.

**Reply by NipulM** [reply]
I'm not quite getting the question. Could you clarify or provide more details so I can better assist you?

**Comment by ghost** [comment]
Are you asking someone to contribute to your page...
